src/Scripting/OverlayBindings.cpp

```diff
@@ -1,6 +1,7 @@
 #include "OverlayBindings.h"
 
 #include "NuklearLegacyBindings.h"
+//#include "ImGuiBindings.h"
```

Comment:
Can drop?

Reply:
Yes, I thought this would have been faster and the next branch could bring in the imguibindings but well :D bad guess